appengine/cloud-tasks/create_http_task.rb

```diff
@@ -0,0 +1,93 @@
+# Copyright 2018 Google, Inc
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloud_tasks_appengine_create_task]
+require "google/cloud/tasks"
+
+# Create an App Engine Task
+#
+# @param [String] project_id Your Google Cloud Project ID.
+# @param [String] location_id Your Google Cloud Project Location ID.
+# @param [String] queue_id Your Google Cloud App Engine Queue ID.
+# @param [String] url The full path to sent the task request to.
+# @param [String] payload The request body of your task.
+# @param [Integer] seconds The delay, in seconds, to process your task.
+def create_http_task project_id, location_id, queue_id, url, payload: nil, seconds: nil
+
+  # Instantiates a client.
+  client = Google::Cloud::Tasks.new
+
+  # Construct the fully qualified queue name.
+  parent = Google::Cloud::Tasks::V2beta3::CloudTasksClient.queue_path project_id, location_id, queue_id
```

Comment:
This PR reminded me, so I opened and released https://github.com/googleapis/google-cloud-ruby/pull/3034. `Google::Cloud::Tasks::V2beta3::CloudTasksClient.queue_path` can now also be accessed via `client.queue_path`